core/trino-main/src/main/java/io/trino/dispatcher/FailedDispatchQuery.java

```diff
@@ -332,6 +332,7 @@ private static QueryStats immediateFailureQueryStats()
                 DataSize.ofBytes(0),
                 0,
                 0,
```

Comment:
For the commits head wrap to 50 characters, body to 72 characters https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md.